package.json

```diff
@@ -1,3041 +1,39 @@
-                        }
-                    },
-                    {
-                        "id": "azDeploy",
-                        "title": "%vscode-docker.walkthrough.dockerStart.azDeploy.title%",
-                        "completionEvents": [
-                            "onCommand:vscode-docker.registries.deployImageToAzure",
-                            "onCommand:vscode-docker.registries.deployImageToAca"
-                        ],
-                        "description": "%vscode-docker.walkthrough.dockerStart.azDeploy.description%",
-                        "when": "isAzureAccountInstalled",
-                        "media": {
-                            "image": "https://github.com/microsoft/vscode-docker/raw/main/resources/walkthroughs/6g-deployToAppSvc.gif",
-                            "altText": "Deploying to Azure App Service"
-                        }
-                    },
-                    {
-                        "id": "learn",
-                        "title": "%vscode-docker.walkthrough.dockerStart.learn.title%",
-                        "description": "%vscode-docker.walkthrough.dockerStart.learn.description%",
-                        "completionEvents": [
-                            "onStepSelected"
-                        ],
-                        "media": {
-                            "markdown": "resources/walkthroughs/empty.md"
-                        }
-                    }
-                ]
-            }
-        ]
-    },
     "engines": {
         "vscode": "^1.92.0"
     },
-    "capabilities": {
-        "virtualWorkspaces": false,
-        "untrustedWorkspaces": {
-            "supported": "limited",
-            "description": "%vscode-docker.workspaceTrust.description%",
-            "restrictedConfigurations": [
-                "docker.commands.build",
-                "docker.commands.run",
-                "docker.commands.runInteractive",
-                "docker.commands.attach",
-                "docker.commands.logs",
-                "docker.commands.composeUp",
-                "docker.commands.composeDown",
-                "docker.environment",
-                "docker.scaffolding.templatePath",
-                "docker.dockerPath",
-                "docker.composeCommand"
-            ]
-        }
-    },
     "scripts": {
-        "watch": "webpack --watch",
-        "build": "webpack --mode production --devtool hidden-source-map",
-        "ci-build": "npm test",
+        "build": "",
         "package": "vsce package",
-        "ci-package": "npm test && vsce package",
-        "lint": "eslint --max-warnings 0 src --ext ts",
-        "test": "node ./out/test/runTest.js",
-        "pretest": "tsc -p ./",
-        "test-watch": "tsc -watch -p ./"
+        "lint": "",
+        "test": ""
     },
     "extensionDependencies": [
-        "vscode.docker",
-        "vscode.yaml"
+        "ms-azuretools.vscode-containers"
```

Comment:
I added Container Tools as a dependency, rather than extension pack. This allows for the Docker Extension Pack to be uninstalled, without uninstalling Container Tools.